histories/elle/list-fail.edn

```diff
@@ -0,0 +1,17 @@
+; P1 and P2 read x and see an empty list
```

Comment:
Bad sample for list-append is already exists, see `histories/elle/list-append-gh-30.edn`.